web-client/src/presenter/computeds/AdvancedSearch/practitionerSearchHelper.ts

```diff
@@ -24,39 +23,40 @@ export type FormattedPractitionerSearchResultType = {
 };
 
 type PractitionerSearchHelperResult = {
-  showNoMatches?: boolean;
-  showSearchResults?: boolean;
-  showStateSelect?: boolean;
-  activePage?: number;
-  formattedSearchResults?: FormattedPractitionerSearchResultType[];
-  numberOfResults?: number;
-  pageCount?: number;
-  pageSize?: number;
-  showPractitionerSearch?: boolean;
+  showNoMatches: boolean;
+  showSearchResults: boolean;
+  activePage: number;
+  formattedSearchResults: FormattedPractitionerSearchResultType[];
+  numberOfResults: string;
+  pageCount: number;
+  pageSize: number;
+  showPaginator: boolean;
 };
 
 export const practitionerSearchHelper = (
   get: Get,
   applicationContext: ClientApplicationContext,
 ): PractitionerSearchHelperResult => {
-  const permissions = get(state.permissions);
-  const countryType: string = get(
-    state.advancedSearchForm.caseSearchByName.countryType,
-  );
   const searchResults = get(state.searchResults['practitioner']);
   const activePage: number = get(
     state.advancedSearchForm.practitionerSearchByName.pageNum,
   );
 
   let result: PractitionerSearchHelperResult = {
     activePage,
-    showPractitionerSearch: permissions?.MANAGE_PRACTITIONER_USERS,
```

Comment:
This is still being referenced and needs to be updated.

Reply:
thanks